modules/core/js/src/main/scala/com/azavea/stac4s/geometry/Geometry.scala

```diff
@@ -0,0 +1,131 @@
+package com.azavea.stac4s.geometry
+
+import cats.Eq
+import cats.syntax.either._
+import cats.syntax.eq._
+import cats.syntax.traverse._
+import io.circe._
+import io.circe.syntax._
+
+sealed abstract class Geometry
+
+object Geometry {
```

Comment:
Eh I guess that's the main cause of 'duplicate' classes implemented separately for the JS.

Reply:
I have an idea but don't know would it work. Can files from the _shared_ have references from classes defined in JVM / JS dirs (depending on the platform)?

Reply:
I don't think you can. To test this, I created an object in `com.azavea.stac4s` in `js/` called `TestObject` then tried to import it in an object in `com.azavea.stac4s` in `shared/` called `TestImport`, and it doesn't work. I think it's a one way street.

Reply:
@jisantuc this is so sad ):

Reply:
Yeah it's pretty messy. I think the duplication probably looks worse than it is because `scala-java-time` mirrors the `java.time` classpath but it's... a lot. I'm not sure what I should add in terms of documentation about development now -- like if you add something with geometry you have to add it to two places, but I'm not sure where that goes in the README